### ISSUE TITLE
This works nicely on the Arduino Giga

### ISSUE THREAD

**Issue by markprince1** [pstolarz/OneWireNg#62]
When I build this, I get a warning that it's not guaranteed on the Arduino Giga R1 board, but I've just built and tested it on both 9 and 12 bit resolution, and it works very well.

Many thanks - that's twice this library has been a lifesaver.

**Comment by pstolarz** [comment]
Yes, the board uses Mbed core and my library bases on generic Mbed API to handle GPIOs, therefore shall work with with all type of Mbed based cores. I will add *mbed_giga* to `library.properties` to get rid of this warning. Thanks for using my lib and if you like it, star it. 😀
